examples/pod_runtime.json

```diff
@@ -18,8 +18,8 @@
                 "user": "0",
                 "mountPoints": [
                     {
-                        "name": "work",
-                        "path": "/mnt/foo"
+                        "name": "database",
```

Comment:
to match https://github.com/appc/spec/blob/cff804693f73e48beb16dab92d149c0a2df3edd3/examples/image.json#L63